### PR TITLE
Tests for ignore config added

### PR DESCRIPTION
**Pull request by sdvendramini**
|Related issue|
|---|
|https://github.com/wazuh/wazuh/issues/17881|

## Description

Migration of ignore cases for FIM integration tests

## Tests

- Compilation without warnings in every supported platform
  - [x] Linux

### CODE REVIEW

File: tests/integration/test_fim/test_files/test_ignore/test_ignore_valid.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        whodata: Implies real-time monitoring but adding the 'who-data' information.
+    - tier:
+        0: Only level 0 tests are performed, they check basic functionalities and are quick to perform.
+        1: Only level 1 tests are performed, they check functionalities of medium complexity.
+        2: Only level 2 tests are performed, they check advanced functionalities and are slow to perform.
+
+tags:
+    - fim_ignore
+'''
+import os
```
Comment (TomasTurina) [inline tests/integration/test_fim/test_files/test_ignore/test_ignore_valid.py:62]: Unused import.

Comment (sdvendramini) [reply tests/integration/test_fim/test_files/test_ignore/test_ignore_valid.py]: Done!

File: tests/integration/test_fim/test_files/test_env_variables/test_cud_events.py
Comment (TomasTurina) [inline tests/integration/test_fim/test_files/test_env_variables/test_cud_events.py:1]: Why is this test empty?

File: tests/integration/test_fim/test_files/test_ignore/test_ignore_valid.py [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+                 are still detected. For this purpose, the test will monitor folders containing files to be ignored
+                 using names or regular expressions. Then it will create these files and check if FIM events should
+                 be generated. Finally, the test will verify that the generated FIM events correspond to the files
+                 that must not be ignored.
+
+    wazuh_min_version: 4.2.0
+
+    tier: 2
+
+    parameters:
```
Comment (TomasTurina) [inline tests/integration/test_fim/test_files/test_ignore/test_ignore_valid.py:110]: Fix description.

File: tests/integration/test_fim/test_files/test_env_variables/test_cud_events.py
Comment (sdvendramini) [reply tests/integration/test_fim/test_files/test_env_variables/test_cud_events.py:1]: Deleted.

File: tests/integration/test_fim/test_files/test_ignore/test_ignore_valid.py (hunk 2 above)
Comment (sdvendramini) [reply tests/integration/test_fim/test_files/test_ignore/test_ignore_valid.py:110]: Done!

**Review by TomasTurina — APPROVED** [review]

**Pull request merged**